pkg/sfu/buffer/buffer.go

```diff
@@ -31,12 +31,14 @@ type pendingPackets struct {
 }
 
 type ExtPacket struct {
-	Head      bool
-	Arrival   int64
-	Packet    rtp.Packet
-	Payload   interface{}
-	KeyFrame  bool
-	RawPacket []byte
+	Head     bool
+	Arrival  int64
+	Packet   rtp.Packet
+	Payload  interface{}
+	KeyFrame bool
+	//audio level for voice, l&0x80 == 0 means audio level not present
```

Comment:
add audio level for inter-sfu transfer, relay nodes can't compute audio level alone, because it don't have information about negotiationed RTPParameters

Reply:
Looks like search and replace issue 😊. It should be 127

Reply:
Maybe worth looking at adding some more bits to the subscribe flow response in addition to what you have now

Reply:
sounds good. more flexible if transfer these information to downstream. we can add a note in linear

Reply:
@cnderrauber MSBit being does not necessarily mean that voice level is not present. It depends on what is negotiated. Probably different browsers behave differently and I know that Chrome has had bugs in it too (I have never seen Chrome set it even if `vad` is negotiated, but they probably have fixed the bug recently). The relevant RFC prose
```
In addition, a flag bit (labeled "V") optionally indicates whether
   the encoder believes the audio packet contains voice activity.  If
   the V bit is in use, the value 1 indicates that the encoder believes
   the audio packet contains voice activity, and the value 0 indicates
   that the encoder believes it does not.  (The voice activity detection
   algorithm is unspecified and left implementation-specific.)  If the V
   bit is not in use, its value is unspecified and MUST be ignored by
   receivers.  The use of the V bit is signaled using the extension
   attribute "vad", discussed in Section 4.
```